src/System.Memory/tests/ParsersAndFormatters/TestException.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+//
+// Intentionally placed in the global namespace so that we don't have to see an annoying long type name
+// in the xunit output.
+//
+internal sealed class TestException : Exception
```

Comment:
This is general purpose and can potentially be used in other tests too. Move this up to src/System.Memory/tests.

Reply:
Done